mongodbatlas/resource_mongodbatlas_database_user_test.go

```diff
@@ -71,7 +73,34 @@ func TestAccResourceMongoDBAtlasDatabaseUser_withX509Type(t *testing.T) {
 					testAccCheckMongoDBAtlasDatabaseUserAttributes(&dbUser, username),
 					resource.TestCheckResourceAttrSet(resourceName, "project_id"),
 					resource.TestCheckResourceAttr(resourceName, "username", username),
-					resource.TestCheckResourceAttr(resourceName, "x509_type", "MANAGED"),
+					resource.TestCheckResourceAttr(resourceName, "x509_type", "CUSTOMER"),
```

Comment:
I would not remove the MANAGED test but add a 2nd test that allows for the username type for CUSTOMER x509 types.  MANAGED is far more common so we should keep the test.

Reply:
No problem, thank you melissa